CONTRIBUTING.md

```diff
@@ -25,30 +25,28 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 
 #### [RISC Zero toolchain](https://dev.risczero.com/api/zkvm/quickstart#1-install-the-risc-zero-toolchain)
 
-The RISC Zero toolchain must match the version of crates you are compiling.
+Run `rzup install` to install the latest Rust and C++ toolchains.
+```bash
+cargo run --bin rzup install rust
+cargo run --bin rzup install cpp
+```
 
-When testing and running examples within this library, you can install `cargo-risczero` from source:
+The versions of `r0vm` and `cargo-risczero` must match the source code. Compile and install these
+from source when using a development branch.
 
 ```bash
 cargo install --force --path risc0/cargo-risczero
+cargo install --force --path risc0/r0vm
```

Comment:
`r0vm` is already included with `cargo-risczero`

Reply:
Ah, I see. updated